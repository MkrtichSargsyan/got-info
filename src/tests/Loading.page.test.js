import renderer from 'react-test-renderer';
import LoadingPage from '../pages/Loading.page';

it('expects to render the loading page', () => {
  const result = renderer.create(<LoadingPage completed={81} />).toJSON;
  expect(result).toMatchSnapshot();
});
