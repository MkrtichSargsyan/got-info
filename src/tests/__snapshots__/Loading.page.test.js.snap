// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render the loading page 1`] = `[Function]`;
